src/components/InstancesList/InstancesList.js

```diff
@@ -922,6 +922,9 @@ class InstancesList extends React.Component {
 
     const onChangeIndex = (e) => {
       this.setState({ optionSelected: e.target.value });
```

Comment:
can be simplified I hope

`const {value} = e.target`
`const isSingleResult = value !== browseModeOptions.CALL_NUMBERS || value !== browseModeOptions.SUBJECTS`
`this.setState({ isSingleResult })`

Reply:
Thanks, @vashjs , I think it will be a better way. I have already fixed it.